docs/installation/windows.md

```diff
@@ -10,7 +10,7 @@ Once downloaded, double-click the installer and follow the prompts.
 There is a shortcut placed in the Start Menu upon installation.
 
 * By default the package is installed to `$env:ProgramFiles\PowerShell\`
-* You can launch PowerShell via the Start Menu or `$env:ProgramFiles\PowerShell\pwsh.exe`
+* You can launch PowerShell via the Start Menu or `$env:ProgramFiles\PowerShell\6.0.0\pwsh.exe`
```

Comment:
Since pwsh is now on the [PATH](https://github.com/PowerShell/PowerShell/blob/master/assets/Product.wxs#L127), we should simplify this to something like `just type pwsh` because this works in any shell (cmd, PowerShell or just typing it into the address bar of explorer)

Reply:
agreed. unless installing from core cmd prompt.  Path in cmd is not updated until new cmd is invoked.

Reply:
Well said, but I think this is standard behaviour and there are tricks to reload the PATH if one really wanted to. If someone installs it via cmd then I would say then they are clever enough to know that. I prefer simplicity in this case but the decision will be up to maintainers.

Reply:
I agree with just using `pwsh` and PATH not being updated is a corner case and pretty standard

Reply:
I would be ok if you added: "...Start Menu or from a new console window `pwsh`.